content/news/2025-06-30-elabftw-integration/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "eLabFTW integration"
+date: "2025-06-30"
```

Comment:
```suggestion
date: "2025-04-01"
```

Reply:
Does this change imply that the post should be published on Wednesday? It makes then sense to add details on the file source template.